scripts/create.py

```diff
@@ -0,0 +1,65 @@
+# #######
+# Copyright (c) 2016 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+'''
+    lifecycle.Create
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Creates the Cloudify Host-Pool Service
```

Comment:
seems kind of misleading, since it basically installs requirements and creates the base directory. It's more of a "pre-configure" or "setup"